pages/common/gh-extension.md

```diff
@@ -22,3 +22,15 @@
 - Upgrade all extensions:
 
 `gh extension upgrade --all`
+
+- List installed extensions:
+
+`gh extension list`
+
+- Remove a extension:
+
+`gh extension remove {{name}}`
```

Comment:
```suggestion
`gh extension remove {{extension_name}}`
```

Reply:
The 1st and 4rd examples use the same `{{name}}` placeholder.

Reply:
I think we should change those too.

Reply:
Most of the time we`d just use `{{extention}}`, like we do for e.g. `{{package}}`. What do you think about that?

Reply:
`{{extension_name}}` or `{{extension}}` would work fine here.